source/Calamari.Tests/KubernetesFixtures/InstallTools.cs

```diff
@@ -133,8 +133,8 @@ public async Task InstallAwsCli()
                 AwsCliExecutable = await DownloadCli("aws",
                     () =>
                     {
-                        var version = "2.8.3";
-                        return Task.FromResult((version, GetAwsCliDownloadLink()));
+                        var version = "latest";
+                        return Task.FromResult((version, GetAwsCliDownloadLink(version)));
```

Comment:
Good point from Yihao was that we want this test to fail if latest aws cli doesn't work with our stuff so I updated this to download the latest but you can optionally change it on your dev machine if you want to test a specific version.

Note: on mac it doesn't install the version it downloads - you need to manually install the version you want to run. 🤷‍♂️

Reply:
it feels like we have a mix of testing low and high ranges

Reply:
I suppose we could fix the version as long as it's noted in the doco.

Reply:
Aren't the other versions fixed for the other tooling?

Reply:
Maybe something to ask others, what is our stance on this?

Reply:
I have fixed the version to what it was meant to be fixed to originally. Can you please give it a final check over and let me know if there are any other issues? 🙏